srcpkgs/fontmatrix/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'fontmatrix'
+pkgname=fontmatrix
+version=0.6.0.20171228
+revision=1
+_commit=8108e6ea8b5944a92d7f27c40509b8e890ddaff1
+wrksrc="${pkgname}-${_commit}"
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="fontconfig-devel freetype-devel MesaLib-devel qt-devel
```

Comment:
I think you can reduce to:
````
makedepends="fontconfig-devel MesaLib-devel qt-webkit-devel"
````

Reply:
Thanks @cr6git, I always forget to run `xmindep`! ;)